Patch by schwardt@univention.de (Fri Sep 11 16:51:37 2009 +0200):

horde: new option to disable "options" link in left sidebar

This patch adds a new config option to hide the "options" link
in oldstyle sidebar and DIMP. The "options" link at the menu on
the top is unaffected.


--- a/horde-webmailer/config/conf.php
+++ b/horde-webmailer/config/conf.php
@@ -79,6 +79,7 @@ $conf['problems']['tickets'] = false;
 $conf['problems']['attachments'] = true;
 $conf['menu']['apps'] = array();
 $conf['menu']['always'] = false;
+$conf['menu']['options_in_sidebar'] = true;
 $conf['menu']['links']['help'] = 'all';
 $conf['menu']['links']['options'] = 'authenticated';
 $conf['menu']['links']['problem'] = 'all';
--- a/horde-webmailer/config/conf.xml
+++ b/horde-webmailer/config/conf.xml
@@ -1510,6 +1510,8 @@
 
    <configboolean name="always" desc="Should we &lt;strong&gt;always
    &lt;/strong&gt; display the Horde frameset?">false</configboolean>
+   <configboolean name="options_in_sidebar" desc="Should we display &quot;options&quot;
+   in left sidebar?">true</configboolean>
 
    <configsection name="links">
     <configenum name="help" desc="Should we display help links in the
--- a/horde-webmailer/services/portal/sidebar.php
+++ b/horde-webmailer/services/portal/sidebar.php
@@ -137,6 +137,7 @@ function buildMenu()
     }
 
     if (Horde::showService('options') &&
+		(isset($conf['menu']['options_in_sidebar']) ? $conf['menu']['options_in_sidebar'] : true) &&
         $conf['prefs']['driver'] != '' && $conf['prefs']['driver'] != 'none') {
         $menu['options'] = array('name' => _("Options"),
                                  'status' => 'active',
